uk.co.compucorp.civicrm.hrleaveandabsences/js/angular/test/shared/components/manager-leave-calendar_test.js

```diff
@@ -25,6 +25,7 @@
       var $componentController, $q, $log, $rootScope, controller, $provide,
         OptionGroup, OptionGroupAPIMock, ContactAPIMock, AbsencePeriod,
         Contact, LeaveRequest, WorkPatternAPI;
+      var isUserAdmin = false;
```

Comment:
Same as above, please leave a comment. Also could you name this var `isAdmin` as well?